src/components/MyRecipesList.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import MyRecipeCard from './MyRecipeCards';
+
+function MyRecipesList({ userRecipes, setUserRecipes }) {
+
+
+    const getAllRecipes = async () => {
+        try {
+            const response = await fetch('http://localhost:3000/recipes/show');
+            if (response.ok) {
+                const data = await response.json();
+                console.log(data);
+                setUserRecipes(data.recipes);
+            }
+        } catch (error) {
+            console.error('Fetch Error:', error);
+        }
+    };
+
+
+    useEffect(() => {
+        getAllRecipes();
+    }, []);
+
+
+    const handleDeleteRecipe = async (recipeId) => {
```

Comment:
i feel like handleDeleteRecipe can be put directly into `MyRecipeCard` component rather than passed in as a prop since the deleting is done there. it would mean that you'll have to pass `setUserRecipe` down to the `MyRecipeCard` as well.